packages/react-components/src/types/ParticipantListParticipant.ts

```diff
@@ -100,7 +100,7 @@ export type ParticipantState =
 /**
  * Audio/Video Media access state
  *
- * @alpha
+ * @public

```

Comment:
nit: this should be `@beta` if this is going to beta. if this is going stable it can be `@public`

Reply:
It is going public.